binding/Binding/SKTypeface.cs

```diff
@@ -16,6 +17,7 @@ public class SKTypeface : SKObject
 		internal SKTypeface (IntPtr handle)
 			: base (handle)
 		{
+			if (handle == IntPtr.Zero) throw new Exception("Invalid typeface");
```

Comment:
Hi,
Indeed, it can be changed to check when you call instead of when you create it, but I am not sure it is the best approach. The reasons I choose to check here are:

1)It implied the lest code changes in the existing codebase. 

2)Conceptually, I think a constructor that takes a handle, should check the handle is valid. The SkData(IntPtr, length) that you mention does that. I think if the need arises to create a managed instance before the native instance is created, the best would be to create a new constructor SKTypeface() and explicitly call that constructor from the methods that need to be created with a null handle, instead of calling the constructor that takes a handle with IntPrt.Zero.

3)I think it is less prone to errors. Here, you test at creation that the handle is invalid, and every method can safely assume the handle is valid. If we move the checks to the callers, we need to call GetObject or check for IntPrt.Zero in each methods that needs to deal with the handle. There are much more calls to change, and the possibility to forget one is higher.

But of course there is no "right" answer, both ways would work. I just wanted to state the reasons why I like it more to check at the constructor, but if you prefer to check later, I can change it, it is no problem.